scylla-cql/src/frame/response/result.rs

```diff
@@ -112,6 +117,52 @@ pub enum CqlValue {
     Varint(CqlVarint),
 }
 
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct DropOptimizedVec<T> {
```

Comment:
@pkolaczk I'm curious, do you have a microbenchmark that suggests this does anything different to a regular `Vec<T>` on release mode?